### PR TITLE
Support to retransmit message

### PR DESCRIPTION
**Pull request by thakurajayL**
Supporting service request message retransmission
This is required when we evaluate 5G core network functon for redundancy events

### CODE REVIEW

**Comment by gab-arrobo** [comment]
@thakurajayL, please resolve conflict

File: profile/context/profile.go [hunk 1]
```diff
@@ -217,6 +218,7 @@ func (profile *Profile) Init() error {
 		return err
 	}
 
+	profile.RetransMsg = false
```
[the remark this answers is not in the document]
Comment (thakurajayL) [reply profile/context/profile.go:221]: Profile 4 is service req and I want to make sure all other profiles have retransMsg is set to false.

[the remark this answers is not in the document]
Comment (gab-arrobo) [reply profile/context/profile.go:221]: The thing is that with this line, you are also changing `profile4` to `false` regardless of what the user has configured for this profile

[the remark this answers is not in the document]
Comment (thakurajayL) [reply profile/context/profile.go]: Understood.. Fixed.

File: profile/context/profile.go [hunk 2]
```diff
@@ -97,6 +97,7 @@ type Profile struct {
 	StepTrigger    bool           `yaml:"stepTrigger" json:"stepTrigger"`
 	StartIteration string         `yaml:"startiteration" json:"startiteration"`
 	Iterations     []*Iterations  `yaml:"iterations"`
+	RetransMsg     bool           `yaml:"retransMsg" json: "retransMsg"`
```
Comment (gab-arrobo) [inline profile/context/profile.go:100]:
```suggestion
	RetransMsg     bool           `yaml:"retransMsg" json:"retransMsg"`
```

**Review by gab-arrobo — CHANGES_REQUESTED** [review]

**Review by gab-arrobo — APPROVED** [review]
+1

**Pull request merged**